<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ceasar">
    <property name="text">
     <string>Ceasar Cipher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rsa">
    <property name="text">
     <string>RSA Cipher</string>
    </property>
   </widget>
  </widget>
  <menuBar class="QMenuBar" name="menubar"/>
  <statusBar class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>